src/i18n/base.js

```diff
@@ -58,7 +58,7 @@ wb.i18nDict = {
 	"srch-menus": "@srch-menus@",
 	email: "@email@",
 	"menu-close": "@menu-close@",
-	"overlay-close": "@overlay-close@",
+	"overlay": "@overlay@",
```

Comment:
To avoid a breaking change, I suggest to keep the same key.

```suggestion
	"overlay-close": "@overlay-close@",
```